s2/bq/doc/REFERENCE.md

```diff
@@ -0,0 +1,162 @@
+## Reference
+
+### S2
+
+#### VERSION
+
+{{% bannerNote type="code" %}}
+s2.VERSION()
+{{%/ bannerNote %}}
+
+Returns the current version of the s2 library.
+
+#### GEOJSONBOUNDARY_FROMKEY
+
+{{% bannerNote type="code" %}}
+s2.GEOJSONBOUNDARY_FROMKEY(key STRING) 
```

Comment:
Do we need geojson from the open API?

Reply:
well, I would say this is not intended to be used directly but from ST_GEOGFROMKEY_BOUNDARY in order to get the the geography. So I will remove it from the Reference